src/test/resources/csv/hollywoodland.thumbs.csv

```diff
@@ -0,0 +1,10 @@
+"Project Name","Item ARK","Parent ARK","Item Status ID","Item Status","Object Type","File Name","Item Sequence","Duplicate","Visibility","Delete in Title","Type.typeOfResource","Subject","Type.genre","Format.medium","Alt ID.local","AltTitle.uniform","Title","Format.dimensions","Publisher.publisherName","Coverage.geographic","Format.extent","Relation.isPartOf","Description.note","Opac url","Finding Aid URL","Bucketeer State","IIIF Access URL","IIIF Manifest URL","thumbnail"
```

Comment:
Noticing only `thumbnail` header is lower case. Perhaps we should upcase to make it consistent with the rest of the headers?

Reply:
Uppercased